### PR TITLE
Update CM_dependencies.cmake

### PR DESCRIPTION
**Pull request by lordofkillz**
changed to automatically detect cuda and cudnn We should update the README.md with.

Visit the NVIDIA cuDNN download page and select the cuDNN version that corresponds to your CUDA version.

After downloading, extract the cuDNN package contents into your CUDA directory, typically located at C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.X. Replace "v11.X" with the actual version number of your CUDA installation.

If prompted to overwrite any existing files during the extraction process, choose "Yes to all" to ensure that the cuDNN files are updated correctly.

### CODE REVIEW

**Comment by stephanecharette** [comment]
The problem I have with this is it goes against the instructions given by NVIDIA.  Nowhere in their instructions do they tell people to exract the contents of CUDNN into the CUDA directory.

File: CM_dependencies.cmake [hunk 1]
```diff
@@ -1,18 +1,23 @@
 # Darknet object detection framework
 
-
 # ==========
 # == CUDA ==
 # ==========
-CHECK_LANGUAGE (CUDA)
```
Comment (stephanecharette) [inline CM_dependencies.cmake:7]: Can you re-do this pull request so it only contains the changed files?

Comment (stephanecharette) [reply CM_dependencies.cmake:7]: I wonder if it is because you deleted line #3 that it somehow thinks every line has changed?

Comment (lordofkillz) [reply CM_dependencies.cmake:7]: ok, and yes, I'm aware it goes against NVidia "recommend" way of doing things but it doesn't create new environment variables it doesn't create new paths and it solves the problems. I have never followed that way it just putting files in a different path. but will I fix it. so, it shows only the changes.

Comment (stephanecharette) [reply CM_dependencies.cmake:7]: The other thing is we need to make sure it doesn't break things on Linux as well.

**Pull request closed without merging**